coreapi/sfdc/src/main/java/com/jalasoft/sfdc/ui/pages/FormBookBaseClassic.java

```diff
@@ -0,0 +1,33 @@
+package com.jalasoft.sfdc.ui.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class FormBookBaseClassic extends FormBoookPage {
+
+    @FindBy(xpath = "//*[@id=\"Name\"]")
+    WebElement viewN;
+    @FindBy(xpath = "//*[@id=\"Description\"]")
+    WebElement viewUName;
+//    @FindBy(xpath ="//*[@id=\"bottomButtonRow\"]/input[1]" )
+//    WebElement saveBtn;
+    public  FormBoookPage clickNewBook(){
+    return new FormBookBaseClassic();    }
+    @Override
+    public FormBoookPage fillDataBook(String nameBook, String descriptionB) {
+        setNameBook(nameBook);
+        setViewDescription(descriptionB);
+        return clickNewBook();
+    }
+
+    private void setNameBook(String nameBook) {
```

Comment:
delete the methods and use the next method:
@override
public PriceBooksDetailPage addNewPriceBooks(String priceBookName, String description) {
driverTools.setInputField(priceBooksName, priceBookName);
driverTools.setInputField(description, description);
return new PriceBooksDetailPageClassic();
}
your methods have to be more descriptive